frames/src/main/java/com/checkout/frames/utils/extensions/CountryExtensions.kt

```diff
@@ -0,0 +1,26 @@
+package com.checkout.frames.utils.extensions
+
+import com.checkout.base.model.Country
+
+/**
+ * The regional indicator symbols are a set of 26 alphabetic Unicode characters (A–Z)
+ * intended to be used to encode ISO 3166-1 alpha-2 two-letter country codes in a way
+ * that allows optional special treatment.
+ *
+ * A pair of regional indicator symbols is referred to as an emoji flag sequence
+ * (although it represents a specific region, not a specific flag for that region).
+ *
+ * 0x1F1E6 - Regional Indicator Symbol Letter A
+ */
+@SuppressWarnings("MagicNumber")
+internal fun Country.emojiFlag(): String {
```

Comment:
Does this function directly have any unit tests ?

Reply:
Discussed on stand-up.